ci/Dockerfile

```diff
@@ -21,9 +21,14 @@ RUN mkdir -p /usr/share/maven /usr/share/maven/ref \
 # Clone the repo.
 RUN mkdir eclipse-ltp \
     && cd eclipse-ltp \
-    && git clone https://github.com/OpenLiberty/liberty-tools-eclipse.git
-
+#    && git clone https://github.com/OpenLiberty/liberty-tools-eclipse.git
+    && git clone https://github.com/scottkurz/liberty-tools-eclipse.git
```

Comment:
It looks to me that all of the changes on this file are all designed for testing and should not be merged.

Reply:
agreed, removed